### DISCUSSION TITLE
Getting SwiftEmitModule normal x86_64 Emitting module for App error

### DISCUSSION THREAD

**Discussion by zeknoss** [ionic-team/capacitor#7372] [Q&A]
Hello,
Since the latest iOS and XCODE updates, I've been getting the error below when I run `npx cap run ios`
```
The following build commands failed:
        SwiftEmitModule normal x86_64 Emitting\ module\ for\ App (in target 'App' from project 'App')
        (1 failure)
```
I deleted the Derived Data contents to no awail.

Any idea why this could be happenning?
Thank you in advance

**Comment by annisaabuhamid** [comment]
im facing the same issue, wonder if got solution on this

**Comment by stnbndr** [comment]
Same here, but only if I use a plugin like the Barcode Scanner. The basic demo app is working.

**Reply by annisaabuhamid** [reply]
are you using the mlkit barcode scanner?

**Reply by stnbndr** [reply]
Yes, this one [@capacitor-mlkit/barcode-scanning](https://capawesome.io/plugins/mlkit/barcode-scanning/). So maybe it's a problem of the plugin? Are you also using this?

**Reply by annisaabuhamid** [reply]
i encounter the same issue too, wonder if you have other plugin for barcode scanner.im already comment on the repo for the plugin it said from the capacitor

**Reply by robingenz** [reply]
If you think the problem is related to `@capacitor-mlkit/barcode-scanning`, please create a bug report in https://github.com/capawesome-team/capacitor-mlkit and  provide a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) using [this template](https://github.com/capawesome-team/.capacitor-app) in a public GitHub repository so I can debug the issue.

**Reply by stnbndr** [reply]
Thank you for your reply. I get the same error while compiling the [mlkit demo app](https://github.com/robingenz/capacitor-mlkit-plugin-demo). So I created a bug report there. Hope this is ok. [https://github.com/robingenz/capacitor-mlkit-plugin-demo/issues/21](https://github.com/robingenz/capacitor-mlkit-plugin-demo/issues/21)
